OpenRA.Mods.Common/Traits/Attack/AttackCharge.cs

```diff
@@ -29,30 +29,33 @@ class AttackChargeInfo : AttackOmniInfo
 		[Desc("Delay between charge attacks (in ticks).")]
 		public readonly int ChargeDelay = 3;
 
+		[Desc("Sound to play when actor charges.")]
+		public readonly string ChargeAudio = null;
+
 		public override object Create(ActorInitializer init) { return new AttackCharge(init.Self, this); }
 	}
 
 	class AttackCharge : AttackOmni, ITick, INotifyAttack, ISync
 	{
-		readonly AttackChargeInfo info;
+		public readonly new AttackChargeInfo Info;
```

Comment:
What earlier discussion? What to use instead? I just did what the compiler told me.